### DISCUSSION TITLE
Reset modal/drawer form fields after submit

### DISCUSSION THREAD

**Discussion by EugeneKostin** [refinedev/refine#1986] [Q&A] [answered]
Hi,
I'm trying to use antd create modal with useModalForm hook. Everything is fine, but after modal submit or close I click on create button again and the form state doesn't reset (also I tried to use modal destroyOnClose prop). I see the same with FineFoods product create drawer. Is it OK or I just missing something in docs?

Thanks everyone in advance for your help

**Comment by omeraplak** [comment, marked as answer]
Hey @EugeneKostin 👋,
Thank you for contacting us! Unfortunately, Ant Design has such a known issue. Maybe you can try Conditional Rendering to skip this issue 👀

like this,
{ visible && <Modal / > }

I hope it will be useful for you 🚀

**Reply by EugeneKostin** [reply]
Thank you for the quick advice!

p.s. less bugs more hugs
